### PR TITLE
Add icons for each section on landing page #426

### PR DESCRIPTION
**Pull request by shreya99oak**
I added the font-awesome icons to the headings, pls review it!

### CODE REVIEW

**Comment by chutten** [comment]
Do you happen to have a live page with this patch? I notice you have this pushed to the gh-pages branch of your fork, is that public? Or can you provide a screenshot, so I can take a look?

**Comment by shreya99oak** [comment]
icons

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script defer src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
```
Comment (chutten) [inline index.html]: We already include the FA CSS in the page. Do we actually need the JS?

**Review by chutten — CHANGES_REQUESTED** [review]
The screenshot looks good, thanks!

File: index.html (hunk 1 above)
Comment (shreya99oak) [reply index.html]: Hi, I tried it without the JS, but it displays empty boxes when you open it on the browser. I'll attach a screenshot..

Comment (shreya99oak) [reply index.html]: without_js

Comment (georgf) [reply index.html]: Going from [the documentation here](https://fontawesome.com/how-to-use/web-fonts-with-css), it should be possible to use font awesome with CSS only.
I might have given the wrong names in #426. We currently use font-awesome 4 on the landing page, but the links are for the font-awesome 5 names. You can try out [the names for fa 5](https://fontawesome.com/v4.7.0/icon/bar-chart/) instead.

Comment (shreya99oak) [reply index.html]: Oh, okay

File: index.html [hunk 2]
```diff
@@ -171,4 +171,4 @@ <h3>What's new</h3>
     <script src="analytics/analytics.js"></script>
     <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 </body>
-</html>
+</html
```
Comment (georgf) [inline index.html]: This change looks unintentional? Lets remove it.

**Review by georgf — COMMENTED** [review]
This looks good, thanks!
There is one small change needed, then this is good to go.

File: index.html (hunk 2 above)
Comment (shreya99oak) [reply index.html]: oh sorry,I'll change that

**Review by georgf — APPROVED** [review]
Thanks, this is a nice improvement!

**Comment by shreya99oak** [comment]
Thanks for the help:) 
Could u pls suggest some bugs to work on ?

**Pull request merged**